JavaScript and Ajax/DeleteStudentRecord.page

```diff
@@ -0,0 +1,45 @@
+<apex:page >
```

Comment:
there are unexpected errors on this page

Reply:
Now on this page not unexpected error